app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for human in people:
+        person = Person(human["name"], human["age"])
+        person_list.append(person)
```

Comment:
Consider using list comprehension to create the `person_list` as suggested in the checklist. This can make the code more concise and efficient.